### DISCUSSION TITLE
Frequency of Kotlin updates

### DISCUSSION THREAD

**Discussion by krzema12** [actions/virtual-environments#5199] [General questions] [answered]
Hi Team,

https://github.com/krzema12/github-actions-kotlin-dsl/issues/122

We currently install Kotlin manually via `snap`, but after we found out it comes preinstalled on the environments, we're considering not installing it manually to save time. **The main question is:** how quickly are new Kotlin versions deployed to `ubuntu-latest` images? Is it a manual thing to do on your side, or it happens automatically?

**Comment by panekj** [comment, marked as answer]
New images are built and deployed every week (with some exceptions) and kotlin is installed from GitHub releases, so as long as there are no issues with it, it will have latest release at the time of build.
https://github.com/actions/virtual-environments/blob/404da76e175a33fc9588a55d35c9be28f8fa42d7/images/linux/scripts/installers/kotlin.sh#L10
